bugbot/rules/assignee_no_login.py

```diff
@@ -123,6 +123,10 @@ def add_action(self, bug):
 
     def handle_bug(self, bug, data):
         bugid = str(bug["id"])
+        if "triage_owner_detail" not in bug:
+            logger.warning("Skip bug %s: no triage owner", bugid)
+            return None
```

Comment:
What if we still unassignee, without involving the triage owner?

Reply:
IDK if the Untriaged components ever had a triage owner, we should maybe assign the "General" triage owner to Untriaged too